taxcalc/utils.py

```diff
@@ -197,14 +196,19 @@ def groupby_income_bins(df, bins=None, right=True):
             indicate (1,2], (2,3], (3,4].
 
     """
-
-    if not bins:
+    if compare_with == "tpc":
```

Comment:
similar comment about webapp settings. Please indicate if one of these pre-defined bins is the desired one for the webapp.